pandas/tests/test_nanops.py

```diff
@@ -1005,6 +1005,23 @@ def test_nanmean(self, tz):
             result = nanops.nanmean(obj)
             assert result == expected
 
+    @pytest.mark.parametrize("dtype", ["M8[ns]", "m8[ns]"])
```

Comment:
can you try with tz aware as well for M8

Reply:
not here since the nanops func is ndarray-only, but i think in the DTA tests is viable

Reply:
kk np

Reply:
getting the test for DTA will require some changes to the DTA method that id like to do in a separate PR